tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/instrumented_assembly_generator_cor_profiler_info.cpp

```diff
@@ -0,0 +1,832 @@
+#include "instrumented_assembly_generator_cor_profiler_info.h"
+#include "instrumented_assembly_generator_helper.h"
+#include "../log.h"
+
+namespace instrumented_assembly_generator
+{
+InstrumentedAssemblyGeneratorCorProfilerInfo::InstrumentedAssemblyGeneratorCorProfilerInfo(
+    IUnknown* pICorProfilerInfoUnk) :
+    m_pICorProfilerInfoUnk(pICorProfilerInfoUnk)
+{
+    Log::Debug("InstrumentedAssemblyGeneratorCorProfilerInfo::.ctor");
+    AddRef();
+}
+
+InstrumentedAssemblyGeneratorCorProfilerInfo::~InstrumentedAssemblyGeneratorCorProfilerInfo()
+{
```

Comment:
no call to release ?

Reply:
Please see [this](https://github.com/DataDog/dd-trace-dotnet/pull/2829#discussion_r902803776) and [this](https://github.com/DataDog/dd-trace-dotnet/pull/2829#discussion_r902788452).
After ebde2f57c8ef03fd167862f981abd5c4958c222a I think it should be ok. WDYT?